jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -6,7 +6,7 @@
         './test-infra/jenkins/bootstrap.py' \
             --bare \
             --job='{job-name}' \
-            --json='{json}' \
+            --json=1 \
```

Comment:
Or just use `--json` ? Both are OK according to #1501 and https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L815-L818